tests/scripts/thread-cert/node.py

```diff
@@ -160,10 +163,14 @@ def _shutdown_docker(self):
             if COVERAGE or OTBR_COVERAGE:
                 self.bash('service otbr-agent stop')
 
-                codecov_cmd = 'bash /tmp/codecov.bash -Z'
                 # Upload OTBR code coverage if OTBR_COVERAGE=1, otherwise OpenThread code coverage.
-                if not OTBR_COVERAGE:
-                    codecov_cmd += ' -R third_party/openthread/repo'
+                if OTBR_COVERAGE:
+                    codecov_cmd = 'bash /tmp/codecov.bash -Z'
```

Comment:
need to fix for OTBR too?

Reply:
It seems OTBR is running the same tests too? If so I think it's good to change it to upload in the same way.

Reply:
Done.